### PR TITLE
first draft of a First Time Users page

### LINKED ISSUE #49: Create content for the 'First time users'
Current content can be found [here](https://grass.osgeo.org/documentation/first-time-users/), but needs to be updated.

### PR DESCRIPTION
**Pull request by mlennert**
Here's a first draft of a First Time Users page to fix #49.
I've selected some specific resources which I personally find particularly useful for newcomers, but others might have different opinions so let me know.
I think that a page on "Basic GRASS GIS concepts" might be useful which could be linked to from the newcomer page. What do you think ?

### CODE REVIEW

**Comment by mlennert** [comment]
Also, some illustrations are needed. Does anyone have a picture of a workshop in action with people learning GRASS GIS ?

**Comment by landam** [comment]
> ne have a picture of a workshop in action with people learning GRASS GIS ?

eg.

https://training.gismentors.eu/grass-gis-irsae-winter-course-2018/gallery.html

We would probably need to ask organizers (@ninsbl) if we could use some of them.

**Comment by landam** [comment]
> > ne have a picture of a workshop in action with people learning GRASS GIS ?
> 
> eg.
> 
> https://training.gismentors.eu/grass-gis-irsae-winter-course-2018/gallery.html
> 
> We would probably need to ask organizers (@ninsbl) if we could use some of them.

Good candidate: https://training.gismentors.eu/grass-gis-irsae-winter-course-2018/_images/DSC_0040.JPG

File: content/learn/newcomers.md [hunk 1]
```diff
@@ -4,6 +4,29 @@ date: 2018-12-29T11:02:05+06:00
...
+* GRASS GIS is extremely powerful and the more you understand some of the basic concepts behing GRASS GIS, the more you will be able to harness that power. It is thus worth spending a bit of time to get familiar with these concepts. Payback will be manifold !
+
+
+### Newcomer tutorials
+Depending on your favorite medium for learning, different entry doors exist into the marvellous world of GRASS GIS.
+
+#### Written
+You can find many tutorials in different languages on the [tutorials](/learn/tutorials) page. But here are some links to specific resources that might help you get started:
+
+* The [Quickstart](https://grass.osgeo.org/grass78/manuals/helptext.html) in the manual pages.
```
Comment (neteler) [inline content/learn/newcomers.md]:
```suggestion
* The [Quickstart](https://grass.osgeo.org/grass-stable/manuals/helptext.html) in the manual pages.
```

(as we now have a redirect)

File: content/learn/newcomers.md [hunk 2]
```diff
@@ -4,6 +4,29 @@ date: 2018-12-29T11:02:05+06:00
...
+* The [Quick GUI tutorial](https://grasswiki.osgeo.org/wiki/Quick_wxGUI_tutorial)
+* An [introductory workshop](http://ncsu-geoforall-lab.github.io/grass-intro-workshop/) from Vaclav Petras & the NCSU GeoForAll Lab.
+* The material from the workshop ["From GRASS GIS novice to power user"](https://grasswiki.osgeo.org/wiki/From_GRASS_GIS_novice_to_power_user_(workshop_at_FOSS4G_Boston_2017))
+* For more [specific topics](https://grass.osgeo.org/grass78/manuals/graphical_index.html) (e.g. raster data, vector data, database, etc), you should also read the relevant topic introductions provided as they will provide you with helpful information about how these specific domains are handled in GRASS GIS.
```
Comment (neteler) [inline content/learn/newcomers.md]:
```suggestion
* For more [specific topics](https://grass.osgeo.org/grass-stable/manuals/graphical_index.html) (e.g. raster data, vector data, database, etc), you should also read the relevant topic introductions provided as they will provide you with helpful information about how these specific domains are handled in GRASS GIS.
```

**Comment by mlennert** [comment]
> > > ne have a picture of a workshop in action with people learning GRASS GIS ?
> > 
> > 
> > eg.
> > https://training.gismentors.eu/grass-gis-irsae-winter-course-2018/gallery.html
> > We would probably need to ask organizers (@ninsbl) if we could use some of them.
> 
> Good candidate: https://training.gismentors.eu/grass-gis-irsae-winter-course-2018/_images/DSC_0040.JPG

Do we only have to ask the organizer or also the participants. The ideal picture to avoid this issue would be one where you see participants from the back ;-)

**Comment by mlennert** [comment]
Merging this in its current state in order to have a first version in the website. Will come back with improvements.

**Comment by mlennert** [comment]
See #70 for future improvements of this page

**Pull request merged**